lib/engines/content_object_store/app/controllers/content_object_store/content_block/documents_controller.rb

```diff
@@ -11,4 +11,30 @@ def show
       content_block_document: @content_block_document,
     )
   end
+
+  def new
```

Comment:
I think this comes up against convos we've had before around the relationship of Document/Edition - afaik although Documents are technically like a 'parent' class, in Whitehall everything is driven from Editions Controllers. The `New_Document_Controller` redirects to an Editions Controller once it has the type of Edition - this didn't make sense to me at first, but I've become used to it as there's so much complexity around the publishing workflow... I'd be up for talking more about it

Reply:
How about the `new` action on the DocumentsController shows the schema types, then once it's chosen, we redirect to the EditionsController, which works regardless of if a Document ID is present?

Reply:
yeah that would make sense - there's actually a method that creates the Document if it doesn't exist https://github.com/alphagov/whitehall/blob/a19ec07326c1342fed12fa1ffff0ae76c09089f8/lib/engines/content_object_store/app/models/content_object_store/content_block/edition/documentable.rb#L22

Reply:
I've got this about 90% done, but might belong better in a separate PR